gst/edge/edge_common.h

```diff
@@ -0,0 +1,40 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * Copyright (C) 2022 Samsung Electronics Co., Ltd.
+ *
+ * @file    edge_common.h
+ * @date    01 Aug 2022
+ * @brief   Common functions for edge sink and src
+ * @author  Yechan Choi <yechan9.choi@samsung.com>
+ * @see     http://github.com/nnstreamer/nnstreamer
+ * @bug     No known bugs
+ *
+ */
+#ifndef __GST_EDGE_H__
+#define __GST_EDGE_H__
+
+#include <glib.h>
+#include <gst/gst.h>
+#include "nnstreamer-edge.h"
+
+#ifndef UNUSED
```

Comment:
```UNUSED``` has already defined in 
https://github.com/nnstreamer/nnstreamer/blob/main/gst/nnstreamer/include/nnstreamer_util.h

Reply:
Removed. Thanks,.